### PR TITLE
First pass at json ops in core

### PR DESCRIPTION
**Pull request by ry**
Depends on https://github.com/denoland/deno/pull/7038

Provides Deno.core.jsonOpSync and Deno.core.jsonOpAsync for dispatching "json" ops.

Limitation: Does not properly handle error handling.

### CODE REVIEW

File: core/core_isolate.rs [hunk 1]
```diff
@@ -484,6 +530,25 @@ where
   callback(current_heap_limit, initial_heap_limit)
 }
 
+fn json_err(_err: ErrBox) -> Value {
```
Comment (bartlomieju) [inline core/core_isolate.rs:533]: Actually we might want to use `JsonError` instead - then you can implement `From<ErrBox> for JsonError` and do `map_err(JsonError::from)` in `register_op_json_<>` methods. I can look into that if you want

Comment (ry) [reply core/core_isolate.rs:533]: Feel free to push to this branch - but don't bother if it distracts you I can try this tomorrow

Comment (ry) [reply core/core_isolate.rs]: I think we can avoid defining this for a first pass. PTAL at my update.

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**